.gitignore

```diff
@@ -1,3 +1,4 @@
 *.toml
 remote_settings_uptake_worrying.egg-info
```

Comment:
we could probably remove this I think...

Reply:
And the `*.toml` line above :)